projects/1.16/assets/woot/woot/lang/zh_cn.json

```
@@ -316,9 +326,15 @@
   "guide.woot.perks.efficiency.0": "这个升级插件可以降低工厂的萌生原液使用。更高等级的升级插件可以使萌生原液使用量降低更多。",
   "guide.woot.perks.efficiency.1": "$(li)等级 I - 15%%减少$(li)等级 II - 25%%减少$(li)等级 III - 30%%减少",
   "guide.woot.perks.efficiency.name": "振奋锅炉",
+  "guide.woot.perks.flayed.0": "这个升级插件允许被生物工厂加工的生物被牺牲仪式用来生成 LP。 更高级别的升级插件将增加仪式可以使用的生物生命值",
+  "guide.woot.perks.flayed.1": "$(li)等级 I - 80%%$(li)等级 II - 90%%$(li)等级 III - 100%%",
+  "guide.woot.perks.flayed.name": "劫掠",
   "guide.woot.perks.headless.0": "$(item)从“头”开始$(0)升级插件能够产生头颅掉落。更高的等级提供更高的掉落率",
   "guide.woot.perks.headless.1": "$(li)等级 I - 25%%几率$(li)等级 II - 50%%几率$(li)等级 III - 80%%几率",
   "guide.woot.perks.headless.name": "从“头”开始",
+  "guide.woot.perks.laser.0": "这个升级插件可以生产工业先锋粉的以太气体于掉落物输出口旁的储罐。 更高等级的升级插件将增加气体的生成量。依据默认配置，仅能依靠凋灵生成。 ",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "guide.woot.perks.laser.0": "这个升级插件可以生产工业先锋的以太气体于掉落物输出口旁的储罐。 更高等级的升级插件将增加气体的生成量。依据默认配置，仅能依靠凋灵生成。 ",
```